src/main/java/net/neoforged/neoforge/common/CommonHooks.java

```diff
@@ -1360,4 +1362,30 @@ public static void onChunkUnload(PoiManager poiManager, ChunkAccess chunkAccess)
             poiManager.remove(sectionPosKey);
         }
     }
+
+    /**
+     * Registers a before and after blocks that can oxidize and de-oxidize
+     *
+     * @param before block with less oxidization
+     * @param after block with more oxidization
+     */
+    public static void registerOxidizableBlock(Block before, Block after) {
```

Comment:
I feel this would fit better in WeatheringCopper. Furthermore, as the HashBiMap is not thread-safe, we should document that this should be done on the main thread (e.g. using FMLCommonSetupEvent) to avoid concurrent modification exceptions.